performant-software/Annotation-Studio
# AS #394 - Redesign navbar and add NASSR logo

# What this PR does

1. Adds a link to NASSR in the top bar
2. Redesigns the navbar to prevent it from covering up the page content on smaller screens. This was an existing issue exacerbated by the addition of NASSR to the navbar.

For the navbar redesign, I stuck to the Bootstrap 3.1.1 classes that the rest of AS uses, as well as a few custom styles.

## Before


https://user-images.githubusercontent.com/64725469/159041013-759610fa-7484-482d-9984-e64f364df107.mov

## After

https://user-images.githubusercontent.com/64725469/159040623-6f91c201-fd36-4bca-89d7-b404c6ef0388.mov

# How to test

Visit as many parts of the site as possible on various screen sizes and make sure the navbar...

1. Never expands to two lines
2. Always shows "COVE Studio | Tenant Name" on the left
3. Always offers login and other menu options, either on the navbar or in a dropdown inside it

I would give some links to different sections of the site, but I'm not familiar with all that's on there and I don't have access to the document view.
